### PR TITLE
python3 compatibility

### PR DESCRIPTION
**Pull request by noraiz-anwar**
https://openedx.atlassian.net/browse/PROD-771

### CODE REVIEW

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification
+
```
Comment (waheedahmed) [inline openedx.yaml]: Add `owner`.

Comment (noraiz-anwar) [reply openedx.yaml]: I am not sure about the owner of this? Is there any way of finding that out?
Thanks.

**Comment by noraiz-anwar** [comment]
@waheedahmed Thanks for having a look. Integrating Pylint and Pep8 sounds a good idea. I'll surely discuss this with the reporter (Farhana) of the ticket to see if this integration can be covered in the scope of current work.
Thanks again.

File: openedx.yaml (hunk 1 above)
Comment (waheedahmed) [reply openedx.yaml]: @noraiz-anwar I think `pmitros` but not sure otherwise you can just add `unknown`.

Comment (nedbat) [reply openedx.yaml]: Definitely not "pmitros".

**Pull request merged**